samples/sensor/icm42605/src/main.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2020 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+#include <stdio.h>
+
+enum {
+	GYRO_FS_2000DPS = 0,
+	GYRO_FS_1000DPS,
+	GYRO_FS_500DPS,
+	GYRO_FS_250DPS,
+	GYRO_FS_125DPS,
+	GYRO_FS_62DPS,
+	GYRO_FS_32DPS,
+	GYRO_FS_15DPS,
+};
+
+enum {
+	ACCEL_FS_16G = 0,
+	ACCEL_FS_8G,
+	ACCEL_FS_4G,
+	ACCEL_FS_2G,
+};
```

Comment:
These enumerations should be provided, e.g. in a public header here https://github.com/zephyrproject-rtos/zephyr/tree/master/include/drivers/sensor

Reply:
Why is sample removed?

Reply:
Current pull request is related with Sensor Driver.
So I removed sample application.
I will commit sample application after this driver is merged.

Reply:
It's ok to add sample application here, as soon as it has its own commit.

Reply:
OK. I will add sample application with new commit soon.
(Also add some commit and will squash. :) )

Reply:
Committed sample application.